### ISSUE TITLE
Provide examples of usage in JSON-LD

### ISSUE THREAD

**Issue by cmungall** [JervenBolleman/FALDO#33]
unfortunately a large fraction of the bioinformatics community will be put off by RDF.

Luckily it's very easy to use JSON-LD as a trojan horse here. The bioinformatics community needs a JSON successor to GFF3 and it would be easy to place FALDO in this place

**Comment by JervenBolleman** [comment]
That makes sense, do you suggest doing this in the README.md? and also should we have a default context at biohackathon.org?

**Comment by JervenBolleman** [comment]
The fig9. in the paper is in JSON-LD

**Comment by cmungall** [comment]
On 22 Aug 2017, at 23:25, JervenBolleman wrote:

> That makes sense, do you suggest doing this in the README.md?

Ideally one example in the README and then some additional examples / 
unit tests elsewhere... maybe even a readthedocs for FALDO. But assuming 
not infinite time to work on this, yes just the README.md for now

> and also should we have a default context at biohackathon.org?

Great idea.

How do things currently get published there? I don't see a gh-pages or 
docs folder.

**Comment by cmungall** [comment]
> The fig9. in the paper is in JSON-LD

Ah yes great, we can just dupe that for now

**Comment by JervenBolleman** [comment]
Regarding a degault context at biohackathon.org, it currently checks out the codebase every hour if I recall @ktym is the maintainer. I think it something we should be able to do something in Japan. I will at least prepare some code to generate an context.

I was looking into having tabs with examples in the README. Otherwise we can do more regarding documentation.

**Comment by cmungall** [comment]
Great start! As you say, we can use additional features of json-ld to make it more appealing to developers: we can have the `@id` field in the main body simply be `id` (it's possible to push all the `@` stuff into the context header or linked document).

I note also this example creates blank nodes for the Positions. I think this is OK, and we can imagine skolemizing or UUIDing as part of a separate translation step if this is perceived to be an issue.

**Comment by cmungall** [comment]
May want to coordinate the json-ld example with bioschemas, see:

https://github.com/BioSchemas/specifications/issues/210

@JervenBolleman your example above is probably too underspecified, we would need at least a reference

**Comment by JervenBolleman** [comment]
@cmungall yes, the example is completely underspecified as I was thinking of some of the existing public JSON for genomic/protein information.  Which are like
```json
{
    "features": [ {"type": "SIGNAL",
        "category": "MOLECULE_PROCESSING",
        "begin": "1",
        "end": "17"}
    ]
}
```
Which are common but put 3 things into one object making it hard to separate them out with just the JSON-LD processing (Maybe framing can help but I am not an expert there).

**Comment by cmungall** [comment]
I don't think JSON-LD processing is sufficient to transform the above into a FALDO-compliant RDF graph (in which each position is its own node).

One possibility is to make FALDO more lenient and allow flatter representations, and define some kind of rule-based transform to entail the deeper representation.
